int64 joint_0
int64 joint_1
int64 joint_2
int64 joint_4
int64 joint_5
---
int64 result